server/scripts/migrate_gcp_tokens_to_connections.py

```
@@ -0,0 +1,241 @@
+left intact so the system stays bilingual during the transition.
+
+# TODO(2026-Q3): remove user_tokens.gcp legacy storage after one release.
+"""
+from __future__ import annotations
+
+import argparse
+import json
+import logging
+import os
+import sys
+from typing import Any, Dict, List, Optional, Tuple
```

Ruff (0.15.14)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)